create / port missing configs for srze

Whennya opened the pull request:
Adds configs for:
- ze_avalanche_reboot_beta7
- ze_conflict_v4_s2
- ze_ffxii_ridorana_cataract_b6
- ze_otakuroom_v5_6fs
- ze_serpentis_temple_v2a
- ze_2049_css1
- ze_ffxii_mt_bur_omisace_v6_beta6fix
- ze_randomizer_c1s4
- ze_terraria_v3_2
- ze_luciddreams_v3_6_f2
- ze_light_shadow_l1_4
- ze_immortal_flame_v1_2
- ze_frozentemple_b8_2
- ze_flower_a1_s4
- ze_system_v1
- ze_minecraft_universe_v1_5s
- ze_minecraft_journey_v1_test_2
- ze_minecraft_adventure_v1_2c
- ze_666_crazy_escape_v2_css2f
- ze_treecolada_v1_3
- ze_ut2004_convoy_v2_2_1
- ze_xenovation_v3_5_s
- ze_uyuni_v1_6_s2
- ze_the_last_chapter_v1_5_css
- ze_thousand_steps_b7c
- ze_three_eyes_v0_7s_fix
- ze_tibia_v4_1s3
- ze_timesink_css1
- ze_tkara_v1
- ze_totemo_roka_b5s
- ze_trials_v1fs
- ze_tyranny2_v1fix
- ze_valkvang_s1
- ze_aztecorum_b3
- ze_bathroom_v2_5s
- ze_binary_z4_s1
- ze_biohazard2_rpd_v4e_004
- ze_biohazard2_sewer_v4c
- ze_bioharzard3_nemesis_b5_2s
- ze_8bit_v5b.cfg
- ze_abandoned_project_v1_2.cfg
- ze_azathoth_v1_css.cfg
- ze_surf_happy_2_newbie_village_css.cfg
- ze_surf_happy_b7s2.cfg
- ze_verdant_v1.cfg

Fixes
- ze_offliner_v2b.cfg mis-format on "behemoth-remix.mp3"	
- ze_m0w0m_b2 ( already on live server but not updated here )

Incomplete 
- ze_luigismanison_fix2

Whennya committed:
import from work repo

Add
- Bunch of cfgs
Fix
- Offliner misformat

Whennya committed:
add

Add
- ze_system_v1
- ze_minecraft_universe_v1_5s
- ze_minecraft_journey_v1_test_2
- ze_minecraft_adventure_v1_2c

Whennya committed:
add

Add
- ze_terraria_v3_2
- ze_luciddreams_v3_6_f2
- ze_light_shadow_l1_4
- ze_immortal_flame_v1_2
- ze_frozentemple_b8_2
- ze_flower_a1_s4
Fix
- ze_system_v1

Whennya committed:
Update ze_666_crazy_escape_v2_css2f.cfg

Whennya committed:
add

Add
- ze_conflict_v4_s2
- ze_ffxii_ridorana_cataract_b6
- ze_otakuroom_v5_6fs
- ze_serpentis_temple_v2a
Incomplete
- ze_luigismanison_fix2

Whennya committed:
add

- ze_2049_css1
- ze_ffxii_mt_bur_omisace_v6_beta6fix
- ze_randomizer_c1s4

Whennya committed:
Update ze_666_crazy_escape_v2_css2f.cfg

Whennya committed:
fix formatting on some imported configs

Whennya committed:
Update ze_m0w0m_b2.cfg

notkoen commented on musicname/ze_ffxii_mt_bur_omisace_v6_beta6fix.cfg:
Latest version is `ze_ffxii_mt_bur_omisace_v6fix` right now, which has different music.

notkoen commented on musicname/ze_treecolada_v1_3.cfg:
```diff
@@ -0,0 +1,8 @@
+    "rain of amethyst-camellia.mp3"     "Camellia - Rain of Amethyst"
+    "road 96 - on the road by robert parker.mp3"       "Robert Parker - On The Road"
+    "rupert holmes - escape.mp3"        "Rupert Holmes - Escape"
+//    "throat singing meme.mp3"       "file name"
```
Should just remove this line because it's not a song.

notkoen reviewed (commented):
This is a nitpick, but I want the configs to have consistent formatting. Keep indentation at spaces and keep spacing between the file name and music name consistent (2 spaces? 8 spaces?).

It is my fault that I did not explicitly mention that file names need to be lowercase. I've added reviews on configs that had issues, and I'll wait until this gets merged before I update the README for other people.

Good work. I also want the other configs in the repository to be updated with consistent formatting, but that should be a separate PR.

Whennya committed:
fix?

Fix capitalization in files
Tab song names further out for readability, ideally would be done how ze_ffxii_mt_bur_omisace_v6_beta6fix.cfg is done but i don't know stages + other info without being on server

Whennya committed:
Update ze_bathroom_v2_5s.cfg

notkoen commented on musicname/ze_conflict_v4_s2.cfg:
```diff
@@ -0,0 +1,9 @@
+    "y2mate.com - deemo 3.1 - siromaru+cranky - conflict (vila remix).mp3"              "Deemo 3.1 - siromaru+cranky - Conflict (VILA Remix)"
+    "y2mate.com - cranky - old gold.mp3"                                                "Cranky - OLD GOLD"
+    "y2mate.com - [official] black gold - cranky _ tapsonic bold new song.mp3"          "[Official] BLACK GOLD - Cranky | TAPSONIC BOLD New song"
+    "y2mate.com - [mv] conflict  siromaru + cranky.mp3"                                 "[MV] conflict / siromaru + cranky"
```
Don't include extra details such as `[Official]` or `[MV]` in the song name.

notkoen commented on musicname/ze_luciddreams_v3_6_f2.cfg:
```diff
@@ -0,0 +1,7 @@
+{
+    "ahisa - Logical Stimulus.mp3"                          "a_hisa - Logical Stimulus"
+    "artcore avetadeltav & lyvia insensibilitv.mp3"         "AvEtaDeltaV & Lyvia - In_sensibilitv"
+    "artcore-veetacrush.mp3"                                "" // cant source cause source got taken down via dmca
```
Comment out missing song

notkoen commented on musicname/ze_luigismanison_fix2.cfg:
```diff
@@ -0,0 +1,10 @@
+    "music3lmz.mp3"             ""
+    "startmusiclmz.mp3"         ""
+  //"zombiedrunkbar.mp3"        ""
+}
```
Comment out missing songs

notkoen commented on musicname/ze_m0w0m_b2.cfg:
```diff
@@ -1,28 +1,29 @@
+    "bgm_lvl6third.mp3"             "xi - Ancient Relics"
+    "bgm_lvl6boss.mp3"              ""
+    "bgm_lvl6end.mp3"               "Cave Story - Last Battle remixed by BREIS"
 }
```
Comment out missing songs

notkoen commented on musicname/ze_offliner_v2b.cfg:
```diff
@@ -1,7 +1,7 @@
-	"alex-tokyo-rose-sacrifice.mp3"			"Alex & Tokyo Rose - Sacrifice"
+	"behemoth-remix.mp3"			            "GosT - Behemoth (Perturbator remix)"
+	"alex-tokyo-rose-sacrifice.mp3"			    "Alex & Tokyo Rose - Sacrifice"
 }
```
Use spaces for formatting

notkoen commented on musicname/ze_serpentis_temple_v2a.cfg:
```diff
@@ -0,0 +1,22 @@
+    "matrix - why mr anderson.mp3"                          "The Matrix Revolutions - Why Mr. Anderson?"
+    "mgs2 - theme.mp3"                                      "Metal Gear Solid 2 Soundtrack - Main Theme"
+    "mgs3 - theme.mp3"                                      "Metal Gear Solid 3 Soundtrack - Main Theme"
+    "run boy run - instrumental (woodkid).mp3"              "Woodkid - Run Boy Run Instrumental"
```
Nitpick: Should add `OST` for the other songs coming from Matrix, as well as keeping the title consistent (`Matrix reloaded` and `The Matrix Reloaded (OST)`?). Also, add parenthese to instrumental so it doesn't get confusing.

notkoen commented on musicname/ze_system_v1.cfg:
```diff
@@ -0,0 +1,11 @@
+    "dancefloor hero.mp3"                                                   "K-Nu - Dancefloor Hero"
+    "gijensu - implosion (tim tama remix).mp3"                              "Gijensu - Implosion (Tim Tama Remix)"
+    "goreshit - burn this moment into the retina of my eye.mp3"             "Goreshit - Burn this moment into the retina of my eye"
+    "goreshit - fine night.mp3"                                             "Goreshit -  Fine Night"
```
Extra space after `-`

notkoen commented on musicname/ze_trials_v1fs.cfg:
```diff
@@ -0,0 +1,13 @@
+    "ivan_torrent_-_icarus_feat_julie_elven[mp3converter.net].mp3"                          "Ivan Torrent (feat. Julie Elven) - Icarus"
+    "epic trailer music royalty free.mp3"                                                   "Grooving Gecko - Orcs are Coming"
+
+}
```
Remote whitespace

Whennya committed:
remove trailing whitespace

notkoen commented on musicname/ze_uyuni_v1_6_s2.cfg:
```diff
@@ -0,0 +1,18 @@
+    "chase_the_light_t.mp3"                         "Two Steps From Hell - Chase The Light (Amaria)"
+    "dream_of_the_shore_near_another_world.mp3"     "Chrono Cross OST - Dream of the Shore Near Another World"
+    "eternity.mp3"                                  "PIKASONIC - Eternity"
+    "liyuebattle.mp3"                               ""
```
Trim whitespace and comment out missing songs

notkoen commented on musicname/ze_666_crazy_escape_v2_css2f.cfg:
```diff
@@ -0,0 +1,15 @@
+    "nrk2_room3_music.mp3"          "Ragnarok Online OST - Angelica"
+    "nrk2_room4_music.mp3"          "Ragnarok Online OST - Labryinth"
+    "nrk2_room5_music.mp3"          "Ragnarok Online OST - Dark Light"
+    "nrk2_room6_music.mp3"          "『Chaos Drifters』- SawanoHiroyuki[nZk] & Jean-Ken Johnny"
```
`nrk2_room6_music.mp3` doesn't follow the `<Artist> - <Song name>` format.

notkoen commented on musicname/ze_arcana_heart_css.cfg:
Consistent formatting

Whennya replied:
https://github.com/Whennya/Music-Names/tree/consistent-formatting
its done in another branch, ill add it to pr later after this one is done / merged

notkoen replied:
Noted

Whennya committed:
comments

Remove extra whitespace
Lowercase arcana_heart_css
Fix mis formatted music

Whennya committed:
nothing to see here
